R/series.R

```diff
@@ -334,6 +334,12 @@ series <- function(x, series, reeval = TRUE, verbose = TRUE){
     if (reeval){
       reeval.dots <- reeval_dots(x = x$composite, series.short = series.short, verbose = FALSE)
       if (length(reeval.dots) > 0){
+        message("To speed up, extend the `seas()` call (see ?series):")
```

Comment:
It's the same code twice? Can we outfactor?

Reply:
I thought that too but decided against it as the whole function consists of two near identical blocks at the moment. Maybe there is room for some consolidation over the whole thing?
A `dump_call_message` (or similar) helper is of course also an option.

Reply:
No need to make this bigger. I am sure the existing code has a lot of room for improvement. We can do this another day. Let's outfactor the new code for now.